packages/camera/camera_windows/example/pubspec.yaml

```diff
@@ -0,0 +1,29 @@
+name: camera_windows_example
+description: Demonstrates how to use the camera_windows plugin.
+publish_to: 'none'
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+  flutter: ">=2.8.0"
+
+dependencies:
+  camera_platform_interface: ^2.1.2
+  camera_windows:
+    # When depending on this package from a real application you should use:
+    #   camera_windows: ^x.y.z
+    # See https://dart.dev/tools/pub/dependencies#version-constraints
+    # The example app is bundled with the plugin so we use a path dependency on
+    # the parent directory to use the current plugin's version.
+    path: ../
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  integration_test:
+    sdk: flutter
+  pedantic: ^1.10.0
```

Comment:
Nit that I missed before: This should be removed. It's only needed in plugins that are still using legacy analysis options.